SmartDeviceLink_Example/MenuManager.swift

```diff
@@ -0,0 +1,193 @@
+//
+//  MenuManager.swift
+//  SmartDeviceLink-Example-Swift
+//
+//  Created by Nicole on 4/11/18.
+//  Copyright © 2018 smartdevicelink. All rights reserved.
+//
+
+import Foundation
+import SmartDeviceLink
+import SmartDeviceLinkSwift
+
+class MenuManager: NSObject {
```

Comment:
Why doesn't this exist in the Obj-C app?

Reply:
Fixed. Now exists.